src/index.tsx

```diff
@@ -17,6 +19,13 @@ const { store, persistor } = redux;
 
 const queryClient = new QueryClient();
 
+Sentry.init({
+  environment: process.env.NODE_ENV,
+  dsn: process.env.SENTRY_ENV,
```

Comment:
DSNs are perfectly safe to use for the public. What's even better is that a single DSN can effortlessly handle all environments. In my opinion, there's really no necessity to store it as SENTRY_ENV. https://docs.sentry.io/product/sentry-basics/dsn-explainer/